### PR TITLE
Add BLE guide

### PR DESCRIPTION
**Pull request by sigvef**

### CODE REVIEW

File: guides/ble-guide.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Characteristics
+
+> A characteristic is a value used in a service along with properties and configuration information about how the value is accessed and information about how the value is displayed or represented.
+
+A characteristic is basically a 20-byte variable that can be read or written to over the air.
+This is the bread and butter of doing BLE communications.
+Each service can have several characteristics.
+
+Example: A heart rate service may expose two characteristics, one called "heart_rate", and one called "temperature".
```
Comment (stianjensen) [inline guides/ble-guide.md]: A little confusing that the heart rate service exposes a temperature, when we've also introduced a separate temperature service.

Comment (sigvef) [inline guides/ble-guide.md]: Do you have a better suggestion?

Comment (stianjensen) [inline guides/ble-guide.md]: The official heart rate service also exposes body location.
Then there's the thermometer service, with both _measured temperature_ characteristic and _temperature type_ (armpit, ear, finger..) https://developer.bluetooth.org/gatt/characteristics/Pages/CharacteristicViewer.aspx?u=org.bluetooth.characteristic.temperature_type.xml

Comment (sigvef) [inline guides/ble-guide.md]: thanks

**Comment by sigvef** [comment]
ping

**Pull request merged**